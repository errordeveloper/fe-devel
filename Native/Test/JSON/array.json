[]
[43,7.98E+3,"\"]",{"fred":[5,6]}]
