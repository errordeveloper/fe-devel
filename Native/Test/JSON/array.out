ARRAY 0
ARRAY 4
  INTEGER 43
  SCALAR 7980
  STRING 2 '"]'
  OBJECT 1
    STRING 4 'fred'
      ARRAY 2
        INTEGER 5
        INTEGER 6
